{"uuid": "f60bf37a-07d2-4852-a7e0-ee7de9ebe81e", "befores": [{"name": "cookies", "status": "failed", "statusDetails": {"message": "AssertionError: Login form button is not presented\n", "trace": "  File \"/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 890, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/doychevvs/vk/2022-1-QAPYTHON-VK-V-Doychev/homework2/ui/fixtures.py\", line 94, in cookies\n    login_page.should_be_open_login_form_button()\n  File \"/usr/local/lib/python3.8/dist-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/doychevvs/vk/2022-1-QAPYTHON-VK-V-Doychev/homework2/ui/pages/login_page.py\", line 27, in should_be_open_login_form_button\n    assert self.is_element_present(*self.locators.OPEN_LOGIN_FORM_BUTTON),\\\n"}, "steps": [{"name": "Ожидание появления кнопки для открытия формы авторизации", "status": "failed", "statusDetails": {"message": "AssertionError: Login form button is not presented\n", "trace": "  File \"/usr/local/lib/python3.8/dist-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/doychevvs/vk/2022-1-QAPYTHON-VK-V-Doychev/homework2/ui/pages/login_page.py\", line 27, in should_be_open_login_form_button\n    assert self.is_element_present(*self.locators.OPEN_LOGIN_FORM_BUTTON),\\\n"}, "start": 1648649134920, "stop": 1648649225256}], "start": 1648649069385, "stop": 1648649225258}], "start": 1648649069385, "stop": 1648649437635}